otx/algorithms/detection/adapters/mmdet/datasets/pipelines/torchvision2mmdet.py

```diff
@@ -15,62 +16,72 @@
 # import cv2 as cv
 from torchvision import transforms as T
 
+# pylint: disable=
+
 
 @PIPELINES.register_module()
 class ColorJitter(T.ColorJitter):
-    """MMDet adapter"""
+    """MMDet adapter."""
 
-    def __init__(self, key_maps=[("img", "img")], **kwargs):
+    def __init__(self, key_maps=None, **kwargs):
```

Comment:
I'm just wondering why we need this code changes here. Is it related with torch version upgrade or mmdet version updrade? How do you find errors in the previous code?

Reply:
This is how pylint errors are handled.. I'm not sure if this causes any problems.